src/polymer/polymer2-element-scanner.ts

```diff
@@ -0,0 +1,271 @@
+/**
+ * @license
+ * Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as estree from 'estree';
+
+import * as astValue from '../javascript/ast-value';
+import {getIdentifierName} from '../javascript/ast-value';
+import {Visitor} from '../javascript/estree-visitor';
+import * as esutil from '../javascript/esutil';
+import {getPropertyValue} from '../javascript/esutil';
+import {JavaScriptDocument} from '../javascript/javascript-document';
+import {JavaScriptScanner} from '../javascript/javascript-scanner';
+import * as jsdoc from '../javascript/jsdoc';
+import {ScannedElement, ScannedFeature} from '../model/model';
+
+
+
+import {analyzeProperties} from './analyze-properties';
+import {ScannedPolymerElement, ScannedPolymerProperty, Options as PolymerElementOptions} from './polymer-element';
+
+export interface ScannedAttribute extends ScannedFeature {
+  name: string;
+  type?: string;
+}
+
+export class Polymer2ElementScanner implements JavaScriptScanner {
```

Comment:
Should we name the old stuff `Polymer1` instead?

Or maybe we'd do it by the import directory? `polymer/v1/element-scanner.ts`?